src/helpers/formatPool.ts

```diff
@@ -19,17 +15,30 @@ type Asset = {
   };
 };
 
-const addAssetCode = (items: VolumeAsset[]) =>
+const addAssetCode = (items: LiquidityPoolAssetInterval[]) =>
   items.map((i) => ({
     assetCode: getAssetCode(i.asset, "-"),
     ...i,
   }));
 
+const convertObjValToLumens = (item: LiquidityPoolAssetInterval) => {
+  const formattedItem = {} as {
+    [key: string]: any;
+  } & LiquidityPoolAssetInterval;
+  Object.entries(item).forEach(([k, v]) => {
+    formattedItem[k] = typeof v === "number" ? fromStroopsToLumen(v) : v;
```

Comment:
this is a little generic to handle a situation where stellar.expert adds more denominations (12h, 10d, 14d, etc)